rascaline/src/calculators/lode/spherical_expansion.rs

```diff
@@ -46,6 +46,10 @@ pub struct LodeSphericalExpansionParameters {
     pub max_angular: usize,
     /// Width of the atom-centered gaussian used to create the atomic density.
     pub atomic_gaussian_width: f64,
+    /// Weight of the center atom contribution to the features.
+    /// If `1.0` the center atom contribution is weighted the same as any other
+    /// contribution.
```

Comment:
Maybe add something about "this is only for the central atom in the central image ..."